website/docs/r/servicebus_queue.html.markdown

```diff
@@ -48,7 +48,7 @@ The following arguments are supported:
 * `namespace_name` - (Required) The name of the ServiceBus Namespace to create
     this queue in. Changing this forces a new resource to be created.
 
-* `location` - (Required) Specifies the supported Azure location where the resource exists.
+* `location` - (Optional / **Deprecated**) Specifies the supported Azure location where the resource exists.
```

Comment:
Maybe add the bit about it being removed in v2.0 of the provider?

Reply:
this is detailed in the 2.0 guide, which will be inbound shortly, so I think this is fine to leave for the moment